.ci/scripts/release/prepare.sh

```diff
@@ -14,7 +14,7 @@ git remote add origin "https://${GITHUB_TOKEN_USR}:${GITHUB_TOKEN_PSW}@github.co
 
 # configure Jenkins GitHub user for Maven container
 git config --global user.email "ci@camunda.com"
-git config --global user.name "${GITHUB_TOKEN_USR}"
```

Comment:
Did you also whitelisted that with infra ?

Reply:
This was infra's suggestion, actually. You can check the linked Slack message for more.